settings.gradle

```diff
@@ -6,7 +6,7 @@ rootProject.name = 'tony'
  */
 rootProject.name = 'tony'
 
-def modules = ['tony-azkaban', 'tony-core', 'tony-cli', 'tony-mini', 'tony-proxy', 'tony-history-server']
```

Comment:
Can you try adding back the `tony-azkaban` module after adding the Guava and Guice exclusion rules I mentioned in my other comment? The Azkaban support is important for us.

Reply:
I will definitely give that a try.
Do you need hadoop 2.7 with azkaban though?

Reply:
Azkaban should work/compile with any Hadoop version (either 2.7 or 2.9). For consistency, it'd be great if all the submodules in TonY can be built with the same Hadoop version.